ngraph/core/src/op/tensor_iterator.cpp

```diff
@@ -168,9 +168,6 @@ void op::v0::TensorIterator::validate_and_infer_types()
 
             auto body_param_partial_shape = body_parameter->get_partial_shape();
             auto input_partial_shape = inputs().at(index).get_source_output().get_partial_shape();
-            NODE_VALIDATION_CHECK(this,
```

Comment:
Do we have tests which are proofed that these checks are not required?

Reply:
This check has already deleted for Loop op, it's just alignment. The check brokes reshape() feature in some specific cases with TI.